### ISSUE TITLE
Bundler can't load v0.3.1 or v0.4.0 

### ISSUE THREAD

**Issue by sbleon** [sonnym/js-asset_paths#14]
Test script:
```
#!/usr/bin/env ruby
require 'bundler/inline'

gemfile true do
  source 'https://rubygems.org'
  gem 'js-asset_paths', '0.4.0'
end
```
Version 0.3.0 works, but 0.3.1 and 0.4.0 do not. The error is:

```/Users/leon/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/js-asset_paths-0.4.0/lib/js-asset_paths.rb:1:in `require': cannot load such file -- js_asset_paths/version (LoadError)```

**Comment by sonnym** [comment]
When I tried this with `gem 'js-asset_paths', path: '.'`, I got different, albeit similar, results.  Does this branch fix the problem?  https://github.com/sonnym/js-asset_paths/tree/bundler_gem_require_fix

**Comment by sbleon** [comment]
That fixes my problem. Thanks!

I was curious about whether the engine could `require` less code. I played
around and found that the following works for me instead of `require
'rails'` in `/lib/js_asset_paths/engine.rb`. I'm not really sure what the
benefit is to including less code, or if my approach makes maintenance
harder because the interdependencies between parts of Rails may change in
the future.
```
require 'action_dispatch'
require 'rails/engine'
```

**Comment by sonnym** [comment]
Good call - while I doubt this will ever be used outside a full rails environment, may as well be more specific.  I have released v0.3.2 and v0.4.1 with these fixes.  Thanks for reporting this!
